style.css

```diff
@@ -1 +1,148 @@
 /*  Add your styling here */
+body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+}
+.headerheader-wrapper {
+    display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+    
+}
+.background  {
+    background-image: url("https://c4.wallpaperflare.com/wallpaper/953/450/507/nuts-cream-dessert-cakes-wallpaper-preview.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 582.4px;
+    background-position: center bottom;
+    overflow: auto;
+}
+
+
+.logo {
+    margin-top: 15px;
+    position: relative;
+    display: flex;
+    justify-content: center;
+
+
+  }
+  .logo > img {
+    
+    height: 150px;
+    flex-direction: column;
+    text-align: center;
+    margin-bottom: 20px;
+  }
+  nav > ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    font-family: 'Roboto', sans-serif;
+    font-size: 16px;
+    margin-top: 25px;
+    color: #838994;
+    padding-left: 0mm;
+    
+  
+  }
+  li {
+    list-style: none;
+    
+  }
+  li:hover {
+	cursor: pointer;
+	color: #282a85;
+}
+.btn {
+	width: 33.33%;
+	border-radius: 4px;
+    color: #ffffff;
+    background-color: #4794f8;
+    font-size: 16px;
+    display: block;
+    padding: 16px 32px;
+    margin: 0 auto;
+    text-decoration: none;
+    text-align: center;
+}
+
+ .btn:hover {
+	cursor: pointer;
+    background-color: #d9400e;
+    text-decoration: none;
+}
+
+.row {
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    justify-content: space-around;
+    padding: 0 10px;
+   
+}
+.col-12{
+    width: 100%;
+    padding: 20px 10px;
+    margin-top: 10px;
+
+}
+.col-12 img{
+    width: 100%;
+}
+.col-6{
+    width: 35%;
+    padding: 20px 10px;
+    margin-top: 10px;
+
+}
+.col-6 img{
+    width: 100% 
+}
+.footer {
+	text-align: center;
+	color: #000000;
+	
+}
+.footer hr {
+	border: 1px solid #eaebec;
+}
+.footer h5 {
+	font-size: 16px;
+	margin-top: 25px;
+}
+
+.footer p {
+	font-size: 15px;
+	font-weight: 200;
+	color: #838994;
+	
+}
+.social li {
+	display: inline-block;
+	margin: 16px .80px;
+	padding: 12px;
+	height: 48px;
+	width: 48px;
+	border: 2px solid #2b4f72;
+	border-radius: 50%;
+}
+@media (min-width: 700px) {
+    .md-col-3{
+
+    width: 22%;
+
+    }
+    .md-col-12{
+        width: 40%;
+    }
+}
```

Comment:
Remove these as well once you've added the other grid. Otherwise this CSS will clash with the grid.